aiosip/call.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from aiohttp import multidict
```

Comment:
With this line, I remembered that I've completely forgotten to merge your previous Pull Request: https://github.com/Eyepea/aiosip/pull/3
Could you rebase on master branch and use multidict library directly everywhere ?